moveit_simple/include/moveit_simple/robot.h

```diff
@@ -557,10 +583,11 @@ class Robot
   virtual std::vector<double> getJointState(void) const;
 
   bool getIK(const Eigen::Affine3d pose, const std::vector<double> &seed,
-    std::vector<double> &joint_point, double timeout = 1, unsigned int attempts = 1) const;
+    std::vector<double> &joint_point, double timeout = 1, unsigned int attempts = 1,
+    bool limit_joint_windup = false) const;
 
   bool getIK(const Eigen::Affine3d pose, std::vector<double> &joint_point,
-    double timeout = 1, unsigned int attempts = 1) const;
+    double timeout = 1, unsigned int attempts = 1, bool limit_joint_windup = false) const;
```

Comment:
``limit_joint_windup`` - i like this level of API visibility for top level users

Reply:
Yes, `getIK()` is developer level so I made it optional here, but `getJointSolution()` is user level and the value passed to `limitJointWindup()` is used.